pkgs/development/guile-modules/guile-git/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl, guile, libgit2
+, autoreconfHook, pkg-config, texinfo
+}:
+
+let
+  bytestructures = stdenv.mkDerivation rec {
+    pname = "scheme-bytestructures";
+    version = "1.0.7";
+    src = fetchurl {
```

Comment:
Is there a reason you don't use `fetchFromGitHub` here?

Reply:
Belligerent laziness